frontend/package.json

```diff
@@ -42,10 +42,8 @@
     "framer-motion": "^10.16.4",
     "fs": "^0.0.1-security",
```

Comment:
Just curious, but is this supposed to be here as well? I don't think we use any code directly from this package haha.